plugin-server/src/utils/db/db.ts

```diff
@@ -529,6 +540,123 @@ export class DB {
         return client ? kafkaMessages : updatedPerson
     }
 
+    private async updatePersonSetProperties(person: Person, client: PoolClient): Promise<void> {
+        // Important: this should always be ran in a transaction with fetchPerson forUpdate=true
```

Comment:
🤔 if this is the case, maybe we should inline it instead of relying on a comment to gatekeep it.

Reply:
I suspected I'll want to use it for merging people as well, but could inline